bindgen-tests/tests/headers/layout_array.h

```diff
@@ -1,4 +1,4 @@
-// bindgen-flags: --with-derive-hash --with-derive-partialeq --with-derive-eq --impl-partialeq --rust-target 1.40
+// bindgen-flags: --with-derive-hash --with-derive-partialeq --with-derive-eq --impl-partialeq --rust-target 1.40 --blocklist-function "__rte_aligned"
```

Comment:
This is a C attribute macro, so produced a useless Rust macro.

I'm not sure if I added this to keep the diff smaller or if it produces wrong Rust code.

Reply:
Yeah, it produces this useless macro which cannot be called since `__attribute__` and `__alligned__` don't exist:

```rust
#[doc(hidden)]
#[macro_export]
macro_rules! __cmacro____rte_aligned {
    ($ a : expr) => {
        __attribute__((__aligned__(($a).into())).into())
    };
}
```